pkg/controller/dgraph/models/query/queries.go

```diff
@@ -258,6 +258,14 @@ func getQueryForAllGroupsData() string {
 			projectedMemoryCost
 			projectedStorageCost
 			projectedCost
+			lastMonthCPUCost
+			lastMonthMemoryCost
+			lastMonthStorageCost
+			lastMonthCost
+			lastLastMonthCPUCost
+			lastLastMonthMemoryCost
+			lastLastMonthStorageCost
```

Comment:
I would like this renamed. But we can take it up as part of another PR.

Reply:
A few suggestions could be `lastButOneMonthCost`, `secondLastMonthCost`, `monthBeforeLastCost`

Reply:
This requires lot of changes in dgraph queries. Will take up this with the next PR